### DISCUSSION TITLE
Sites (dot yaml) Eloquent

### DISCUSSION THREAD

**Discussion by mwagena** [statamic/cms#10452] [Q&A]
Hi there,

Is it possible (like with the Eloquent driver) to store the sites in a separate location eg database?
If so, I'm curious how to! 

Regards,

Martijn

**Comment by ryanmitchell** [comment]
Not yet but feel free to open an issue on the eloquent driver repo and we can look to add support there.

**Reply by ryanmitchell** [reply]
This is now supported in eloquent driver 4.10.0+
